substrate/frame/support/src/traits/tokens/fungible/mod.rs

```diff
@@ -17,26 +17,108 @@
+//! Also see the [`tokens_in_substrate`] reference docs for more information about the place of
+//! `fungible` traits in Substrate.
+//!
+//! # Avaliable Traits
+//! - [`Inspect`]: Regular balance inspector functions.
+//! - [`Unbalanced`]: Low-level balance mutating functions. Does not guarantee proper book-keeping
+//!   and so should not be called into directly from application code. Other traits depend on this
+//!   and provide default implementations based on it.
+//! - [`UnbalancedHold`]: Low-level balance mutating functions for balances placed on hold. Does not
 //!   guarantee proper book-keeping and so should not be called into directly from application code.
 //!   Other traits depend on this and provide default implementations based on it.
-//! - `Mutate`: Regular balance mutator functions. Pre-implemented using `Unbalanced`, though the
-//!   `done_*` functions should likely be reimplemented in case you want to do something following
-//!   the operation such as emit events.
-//! - `InspectHold`: Inspector functions for balances on hold.
-//! - `MutateHold`: Mutator functions for balances on hold. Mostly pre-implemented using
-//!   `UnbalancedHold`.
-//! - `InspectFreeze`: Inspector functions for frozen balance.
-//! - `MutateFreeze`: Mutator functions for frozen balance.
-//! - `Balanced`: One-sided mutator functions for regular balances, which return imbalance objects
+//! - [`Mutate`]: Regular balance mutator functions. Pre-implemented using [`Unbalanced`], though
+//!   the `done_*` functions should likely be reimplemented in case you want to do something
+//!   following the operation such as emit events.
+//! - [`InspectHold`]: Inspector functions for balances on hold.
+//! - [`MutateHold`]: Mutator functions for balances on hold. Mostly pre-implemented using
+//!   [`UnbalancedHold`].
+//! - [`InspectFreeze`]: Inspector functions for frozen balance.
+//! - [`MutateFreeze`]: Mutator functions for frozen balance.
+//! - [`Balanced`]: One-sided mutator functions for regular balances, which return imbalance objects
 //!   which guarantee eventual book-keeping. May be useful for some sophisticated operations where
 //!   funds must be removed from an account before it is known precisely what should be done with
 //!   them.
+//!
+//! ## Terminology
+//!
+//! - **Total Issuance**: The total number of units in existence in a system.
+//!
+//! - **Total Balance**: The sum of an accounts free and held balances.
+//!
+//! - **Free Balance**: A portion of an accounts total balance that is not held.
+//!
+//! - **Held Balance**: Held balance still belongs to the account holder, but is suspended. It can
+//!   be slashed, but only after all the free balance has been slashed.
+//!
+//!   Multiple holds stack rather than overlay. This means that if an account has
+//!   3 locks for 100 units, the account can spend it's funds for any reason down to 300 units, at
```

Comment:
```suggestion
//!   3 locks for 100 units, the account can spend its funds for any reason down to 300 units, at
```

"it's" is short for "it is", "its" is possessive